src/js/angular/ttyg/controllers/ttyg-view.controller.js

```diff
@@ -633,12 +633,10 @@ function TTYGViewCtrl(
         }
     };
 
-    const onCopiedAnswerToClipboard = (chatItem) => {
-        if (navigator.clipboard) {
-            navigator.clipboard.writeText(chatItem.answer.message)
-                .then(() => TTYGContextService.emit(TTYGEventName.COPY_ANSWER_TO_CLIPBOARD_SUCCESSFUL))
-                .catch(() => TTYGContextService.emit(TTYGEventName.COPY_ANSWER_TO_CLIPBOARD_FAILURE));
-        }
+    const onCopyTextToClipboard = (text) => {
+        navigator.clipboard.writeText(text)
```

Comment:
see the issue which Desislava fixed yesterday https://github.com/Ontotext-AD/graphdb-workbench/pull/1577
Did you consider reusing the copy directive I implemented for the import view?

Reply:
Yes, it was my first choice, but it expects the copy button and the element marked with the 'copyable' class to be within the same container.

Reply:
so? Is that a problem?

Reply:
Done